src/commands/deploy.rs

```diff
@@ -398,6 +415,74 @@ impl DeployCommand {
     }
 }
 
+async fn wait_for_ready(
+    address: &str,
```

Comment:
It's a bit confusing to me to call this "address", which sounds like a `SocketAddr`. "host" perhaps?

Reply:
Changed to `app_domain` which better fits the calling site.